tests/test_0001_html.py

```diff
@@ -2,52 +2,63 @@
 
 import pytest
 
+from lona.html import Div
 
-def test_attribute_dict():
-    from lona.html import Div
 
-    # init ####################################################################
-    d = Div(foo='foo', bar='bar')
+@pytest.mark.incremental()
+class TestAttributeDict:
+    def test_get(self):
+        d = Div(foo='foo', bar='bar')
 
-    assert d.attributes['foo'] == 'foo'
-    assert d.attributes['bar'] == 'bar'
+        assert d.attributes['foo'] == 'foo'
+        assert d.attributes['bar'] == 'bar'
 
-    # reset ###################################################################
-    d = Div()
```

Comment:
@maratori The test seems to be changed here. I don't think this is critical but, this should be a plain refactoring pr that contains no functional changes

Reply:
@fscherf Agree. Fixed.

Should I refactor other tests as well in this PR?

Reply:
@maratori Sure, that would be great! :)

Reply:
Done